systemtest/src/test/java/io/strimzi/systemtest/oauth/OauthAuthorizationST.java

```diff
@@ -61,11 +61,11 @@
     @Description("As a member of team A, I should be able to read and write to all topics starting with a-")
     @Test
     void smokeTestForClients() throws IOException, InterruptedException, ExecutionException, TimeoutException {
-        Future<Integer> producer = teamAOauthKafkaClient.sendMessagesTls(TOPIC_A, NAMESPACE, CLUSTER_NAME, TEAM_A_CLIENT,
-                MESSAGE_COUNT, "SSL");
+        Future<Integer> producer = teamAOauthKafkaClient.sendMessages(TOPIC_A, NAMESPACE, CLUSTER_NAME,
```

Comment:
What is the reason to move so massively to not using TLS? Again, it looks a bit unrelated to the main PR.

Reply:
I realized that it would be better to test just a "authorization" and let the encryption be.

Reply:
Ok, makes sense.